sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/RebaseDateTime.scala

```diff
@@ -66,15 +66,53 @@ object RebaseDateTime {
   // The starting point is the `0001-01-01` (-719164 days since the epoch in
   // Julian calendar). All dates before the staring point have the same difference
   // of 2 days in Julian and Proleptic Gregorian calendars.
```

Comment:
shall we remove `All dates before the staring point have the same difference ...`?

Reply:
For this particular array, it is valid comment. It is not valid for `rebaseJulianToGregorianDays`, strictly speaking.